sample_analyze_tax_us_w2.py

```diff
@@ -0,0 +1,324 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_tax_us_w2.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze US W-2 tax forms.
+
+    See fields found on a US W-2 tax form here:
+    https://aka.ms/azsdk/documentintelligence/taxusw2fieldschema
+
```

Comment:
Same, the short links seem not work. Please double check. Also it's a good idea to put the field schema link in the description section, please apply it to other prebuilt models as well.

Reply:
Already applied it to other prebuilt models